.gitlab-ci.yml

```diff
@@ -0,0 +1,24 @@
+variables:
+  GIT_CLONE_PATH: $CI_BUILDS_DIR/meta-libaktualizr-demo
+
+image: "crops/yocto:ubuntu-18.04-base"
+
+build:
+  script:
+    - cd ..
+    - git clone -b warrior https://git.yoctoproject.org/git/poky
+    - git clone -b warrior https://github.com/openembedded/meta-openembedded.git
+    - source poky/oe-init-build-env
+    - export LC_ALL=en_US.UTF-8
```

Comment:
Our docker image already contains this: https://github.com/advancedtelematic/meta-updater/blob/master/scripts/ci/Dockerfile.bitbake. Is there a reason you're using another third party image?

Reply:
No strong reasons, just wanted to keep it simpler than in meta-updater and see how the [community-proposed](https://git.yoctoproject.org/cgit/cgit.cgi/crops/about/) docker image works.